tuf-spec.md

```diff
@@ -688,9 +688,9 @@ repo](https://github.com/theupdateframework/specification/issues).
 * **4.4. File formats: snapshot.json**
 
    The snapshot.json file is signed by the snapshot role.  It lists the version
-   numbers of all metadata on the repository, excluding timestamp.json and
-   mirrors.json.  For the root role, the hash(es), size, and version number
-   are listed.
+   numbers of all metadata on the repository, excluding root.json, timestamp.json and
```

Comment:
```suggestion
   numbers of only the top-level and all delegated targets role metadata.
```

Reply:
Can we make this even clearer? I read your change as "top-level roles and delegated targets roles", which would mean all roles, which is not what we want. What about...

```suggestion
   numbers of only the top-level targets and all delegated targets role metadata.
```